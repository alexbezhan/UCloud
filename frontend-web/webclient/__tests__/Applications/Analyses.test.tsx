import {createMemoryHistory} from "history";
import "jest-styled-components";
import * as React from "react";
import {Provider} from "react-redux";
import {MemoryRouter} from "react-router";
import {create} from "react-test-renderer";
import {createResponsiveStateReducer, responsiveStoreEnhancer} from "redux-responsive";
import {ThemeProvider} from "styled-components";
import JobResults from "../../app/Applications/JobResults";
import analyses from "../../app/Applications/Redux/AnalysesReducer";
import {initAnalyses} from "../../app/DefaultObjects";
import theme, {responsiveBP} from "../../app/ui-components/theme";
import {configureStore} from "../../app/Utilities/ReduxUtilities";
import {analyses as analysesPage} from "../mock/Analyses";

const configureTestStore = () => configureStore({analyses: initAnalyses()}, {
    analyses,
    responsive: createResponsiveStateReducer(
        responsiveBP,
        {infinity: "xxl"}),
}, responsiveStoreEnhancer);

describe("Analyses component", () => {
    test("Mount component with non-empty page", () => {
        const store = configureTestStore();
        store.getState().analyses.page = analysesPage;
        expect(create(
            <Provider store={store}>
                <ThemeProvider theme={theme}>
                    <MemoryRouter>
                        <JobResults history={createMemoryHistory()} />
                    </MemoryRouter>
                </ThemeProvider>
            </Provider>).toJSON()).toMatchSnapshot();
    });
});
