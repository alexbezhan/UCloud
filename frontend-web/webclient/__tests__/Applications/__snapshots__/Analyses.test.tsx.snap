// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Analyses component Mount component with non-empty page 1`] = `
.c1 {
  cursor: auto;
  margin-left: 190px;
  padding-top: 48px;
  padding-bottom: 14px;
  padding-left: 14px;
  padding-right: 0;
  background-color: #fff;
}

.c4 {
  cursor: auto;
  margin-left: auto;
}

.c13 {
  cursor: auto;
  padding-top: 48px;
}

.c15 {
  cursor: pointer;
}

.c19 {
  cursor: pointer;
  margin-left: -17px;
  padding-left: 15px;
  margin-right: -17px;
  width: auto;
}

.c20 {
  cursor: auto;
  margin-bottom: 16px;
  margin-top: 16px;
}

.c23 {
  cursor: auto;
  margin-bottom: 16px;
}

.c24 {
  cursor: auto;
  padding-top: 48px;
  padding-right: 340px;
}

.c38 {
  cursor: auto;
}

.c3 {
  cursor: inherit;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c6 {
  cursor: pointer;
}

.c7 {
  cursor: inherit;
}

.c10 {
  cursor: pointer;
  margin-left: -17px;
  padding-left: 17px;
  margin-right: -17px;
}

.c14 {
  cursor: inherit;
  font-size: 24px;
  margin: 0;
  font-weight: 400;
}

.c16 {
  cursor: inherit;
  font-size: 24px;
  margin: 0;
  margin-top: 16px;
  font-weight: 400;
}

.c8 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  vertical-align: middle;
  cursor: inherit;
  margin-left: .7em;
}

.c32 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  vertical-align: middle;
  cursor: inherit;
}

.c34 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  vertical-align: middle;
  cursor: pointer;
  margin-right: .4em;
}

.c40 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  vertical-align: middle;
  cursor: inherit;
  margin-right: 8px;
  color: #c00;
}

.c41 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  vertical-align: middle;
  cursor: inherit;
  margin-right: 8px;
  color: #00C05A;
}

.c17 {
  font-size: 10px;
  -webkit-letter-spacing: 0.2px;
  -moz-letter-spacing: 0.2px;
  -ms-letter-spacing: 0.2px;
  letter-spacing: 0.2px;
  display: block;
  margin: 0;
  font-size: 14px;
  color: #000;
  font-weight: 700;
  width: 100%;
}

.c5 {
  position: relative;
  display: inline-block;
}

.c5:hover > div {
  display: block;
}

.c9 {
  overflow: visible;
  border-bottom-left-radius: 5px;
  border-bottom-right-radius: 5px;
  border-top-left-radius: 5px;
  border-top-right-radius: 5px;
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,.14),0px 1px 18px 0px rgba(0,0,0,.12);
  position: absolute;
  background-color: #fff;
  color: #000;
  width: 80px;
  min-width: 80pxpx;
  padding: 12px 16px;
  z-index: 47;
  text-align: left;
  cursor: pointer;
  opacity: 0;
  pointer-events: none;
  left: 85px;
}

.c9 > *:hover:not(.c42) {
  background-color: rgba(0,0,0,0.05);
}

.c9 svg {
  margin-right: 1em;
}

.c9 > svg ~ span {
  margin-right: 1em;
}

.c18 {
  overflow: visible;
  border-bottom-left-radius: 5px;
  border-bottom-right-radius: 5px;
  border-top-left-radius: 5px;
  border-top-right-radius: 5px;
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,.14),0px 1px 18px 0px rgba(0,0,0,.12);
  position: absolute;
  background-color: #fff;
  color: #000;
  width: 138px;
  min-width: 138pxpx;
  padding: 12px 16px;
  z-index: 47;
  text-align: left;
  cursor: pointer;
  opacity: 0;
  pointer-events: none;
}

.c18 > *:hover:not(.c42) {
  background-color: rgba(0,0,0,0.05);
}

.c18 svg {
  margin-right: 1em;
}

.c18 > svg ~ span {
  margin-right: 1em;
}

.c11 {
  cursor: auto;
}

.c25 {
  cursor: auto;
}

.c21 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 5px;
  border-width: 1px;
  border-style: solid;
  border-color: #c9d3df;
}

.c21 > .c0 {
  width: 100%;
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
}

.c21 .c43 {
  border: 0;
  box-shadow: none;
}

.c30 {
  cursor: auto;
  display: inline-block;
  position: relative;
  vertical-align: middle;
  cursor: pointer;
  color: #8393A7;
}

.c30 svg[data-name="checked"] {
  display: none;
}

.c30 > input:checked ~ svg[data-name="checked"] {
  display: inline-block;
  color: #006aff;
}

.c30 > input:checked ~ svg[data-name="empty"] {
  display: none;
}

.c31 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  opacity: 0;
  position: absolute;
}

.c2 {
  cursor: auto;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-left: 204px;
  padding-right: 14px;
  width: 100%;
  height: 48px;
  background-color: #fff;
  position: absolute;
  top: 48px;
  left: 0;
  position: fixed;
  z-index: 50;
}

.c12 {
  cursor: auto;
  padding-top: 96px;
  padding-left: 14px;
  padding-right: 14px;
  width: 340px;
  height: 100%;
  position: absolute;
  top: 0;
  right: 0;
  overflow-y: auto;
  position: fixed;
  z-index: 45;
}

.c22 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: block;
  width: 100%;
  font-family: inherit;
  color: inherit;
  font-size: 14px;
  background-color: transparent;
  border-radius: 5px;
  border-width: 0px;
  border-style: solid;
  border-color: #c9d3df;
  padding-top: 14px;
  padding-bottom: 14px;
  padding-left: 12px;
  padding-right: 12px;
  margin: 0;
}

.c22::-webkit-input-placeholder {
  color: #8393A7;
}

.c22::-moz-placeholder {
  color: #8393A7;
}

.c22:-ms-input-placeholder {
  color: #8393A7;
}

.c22::placeholder {
  color: #8393A7;
}

.c22::-ms-clear {
  display: none;
}

.c26 {
  background-color: #fff;
  border: 0px;
  border-spacing: 0;
  table-layout: fixed;
  width: 100%;
  min-width: 15em;
  background-color: #fff;
}

.c37 {
  border: 0px;
  border-spacing: 0;
  text-align: center;
}

.c39 {
  border: 0px;
  border-spacing: 0;
}

.c28 {
  cursor: auto;
}

.c28 > .c36 {
  border-spacing: 0;
  border-top: 1px solid rgba(34,36,38,.1);
  padding-top: 8px;
  padding-bottom: 8px;
}

.c35 {
  cursor: pointer;
}

.c35 > .c36 {
  border-spacing: 0;
  border-top: 1px solid rgba(34,36,38,.1);
  padding-top: 8px;
  padding-bottom: 8px;
}

.c27 {
  background-color: #fff;
  padding-top: 11px;
  padding-bottom: 11px;
}

.c29 {
  border-spacing: 0;
  border: 0px;
  text-align: center;
  width: 4%;
  background-color: #fff;
  top: 96px;
  z-index: 10;
  position: -webkit-sticky;
  position: sticky;
}

.c33 {
  border-spacing: 0;
  border: 0px;
  text-align: left;
  background-color: #fff;
  cursor: pointer;
  top: 96px;
  z-index: 10;
  position: -webkit-sticky;
  position: sticky;
}

@media screen and (max-width:511px) {
  .c11 {
    display: none;
  }
}

@media screen and (min-width:512px) and (max-width:639px) {
  .c11 {
    display: none;
  }
}

@media screen and (min-width:640px) and (max-width:767px) {
  .c11 {
    display: none;
  }
}

@media screen and (min-width:768px) and (max-width:1023px) {
  .c25 {
    display: none;
  }
}

@media screen and (min-width:1024px) and (max-width:1279px) {
  .c25 {
    display: none;
  }
}

@media screen and (min-width:1280px) {
  .c25 {
    display: none;
  }
}

<div
  className="c0 c1"
  cursor="auto"
>
  <div
    className="c0 c2"
    cursor="auto"
    height={48}
    width={1}
  >
    <div
      className="c3"
      cursor="inherit"
    >
      <div
        className="c0 c4"
        cursor="auto"
      />
      <div
        className="c5"
        data-tag="dropdown"
      >
        <span
          className="c6"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
             
            Jobs per page 25
          </span>
          <svg
            className="c8"
            clipRule="evenodd"
            cursor="inherit"
            fill="currentcolor"
            fillRule="evenodd"
            height=".7em"
            ml=".7em"
            viewBox="0 0 24 12"
            width=".7em"
          >
            <path
              d="M0 0l11.959 12 12-12H0z"
            />
          </svg>
        </span>
        <div
          className="c9"
          color="black"
          cursor="pointer"
          disabled={false}
          onClick={[Function]}
          overflow="visible"
          width="80px"
        >
          <div
            className="c10"
            cursor="pointer"
            onClick={[Function]}
          >
            10
          </div>
          <div
            className="c10"
            cursor="pointer"
            onClick={[Function]}
          >
            25
          </div>
          <div
            className="c10"
            cursor="pointer"
            onClick={[Function]}
          >
            50
          </div>
          <div
            className="c10"
            cursor="pointer"
            onClick={[Function]}
          >
            100
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0 c11"
    cursor="auto"
  >
    <div
      className="c0 c12"
      cursor="auto"
      height="100%"
      width={340}
    >
      <div
        className="c0 c13"
        cursor="auto"
      >
        <h3
          className="c14"
          cursor="inherit"
          fontSize={4}
        >
          Quick Filters
        </h3>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            Today
          </span>
        </div>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            Yesterday
          </span>
        </div>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            This week
          </span>
        </div>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            No filter
          </span>
        </div>
        <h3
          className="c16"
          cursor="inherit"
          fontSize={4}
        >
          Active Filters
        </h3>
        <label
          className="c17"
          color="black"
          fontSize={1}
          fontWeight="bold"
          width="100%"
        >
          Filter by app state
        </label>
        <div
          className="c5"
          data-tag="dropdown"
        >
          <span
            className="c6"
            cursor="pointer"
            onClick={[Function]}
          >
            Don't filter
            <svg
              className="c8"
              clipRule="evenodd"
              cursor="inherit"
              fill="currentcolor"
              fillRule="evenodd"
              height=".7em"
              ml=".7em"
              viewBox="0 0 24 12"
              width=".7em"
            >
              <path
                d="M0 0l11.959 12 12-12H0z"
              />
            </svg>
          </span>
          <div
            className="c18"
            color="black"
            cursor="pointer"
            disabled={false}
            onClick={[Function]}
            overflow="visible"
            width="138px"
          >
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Validated
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Prepared
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Scheduled
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Running
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Transfer success
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Success
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Failure
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Cancelling
            </div>
          </div>
        </div>
        <div
          className="c0 c20"
          cursor="auto"
        >
          <label
            className="c17"
            color="black"
            fontSize={1}
            fontWeight="bold"
            width="100%"
          >
            Job created after
          </label>
          <div
            className="c21"
          >
            <div
              className="react-datepicker-wrapper"
            >
              <div
                className="react-datepicker__input-container"
              >
                <input
                  className="c22"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Don't filter"
                  readOnly={false}
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="c0 c23"
          cursor="auto"
        >
          <label
            className="c17"
            color="black"
            fontSize={1}
            fontWeight="bold"
            width="100%"
          >
            Job created before
          </label>
          <div
            className="c21"
          >
            <div
              className="react-datepicker-wrapper"
            >
              <div
                className="react-datepicker__input-container"
              >
                <input
                  className="c22"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Don't filter"
                  readOnly={false}
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0 c24"
    cursor="auto"
  >
    <div
      className="c0 c25"
      cursor="auto"
    >
      <div
        className="c0 c13"
        cursor="auto"
      >
        <h3
          className="c14"
          cursor="inherit"
          fontSize={4}
        >
          Quick Filters
        </h3>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            Today
          </span>
        </div>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            Yesterday
          </span>
        </div>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            This week
          </span>
        </div>
        <div
          className="c0 c15"
          cursor="pointer"
          onClick={[Function]}
        >
          <span
            className="c7"
            cursor="inherit"
          >
            No filter
          </span>
        </div>
        <h3
          className="c16"
          cursor="inherit"
          fontSize={4}
        >
          Active Filters
        </h3>
        <label
          className="c17"
          color="black"
          fontSize={1}
          fontWeight="bold"
          width="100%"
        >
          Filter by app state
        </label>
        <div
          className="c5"
          data-tag="dropdown"
        >
          <span
            className="c6"
            cursor="pointer"
            onClick={[Function]}
          >
            Don't filter
            <svg
              className="c8"
              clipRule="evenodd"
              cursor="inherit"
              fill="currentcolor"
              fillRule="evenodd"
              height=".7em"
              ml=".7em"
              viewBox="0 0 24 12"
              width=".7em"
            >
              <path
                d="M0 0l11.959 12 12-12H0z"
              />
            </svg>
          </span>
          <div
            className="c18"
            color="black"
            cursor="pointer"
            disabled={false}
            onClick={[Function]}
            overflow="visible"
            width="138px"
          >
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Validated
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Prepared
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Scheduled
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Running
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Transfer success
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Success
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Failure
            </div>
            <div
              className="c0 c19"
              cursor="pointer"
              onClick={[Function]}
              width="auto"
            >
              Cancelling
            </div>
          </div>
        </div>
        <div
          className="c0 c20"
          cursor="auto"
        >
          <label
            className="c17"
            color="black"
            fontSize={1}
            fontWeight="bold"
            width="100%"
          >
            Job created after
          </label>
          <div
            className="c21"
          >
            <div
              className="react-datepicker-wrapper"
            >
              <div
                className="react-datepicker__input-container"
              >
                <input
                  className="c22"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Don't filter"
                  readOnly={false}
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="c0 c23"
          cursor="auto"
        >
          <label
            className="c17"
            color="black"
            fontSize={1}
            fontWeight="bold"
            width="100%"
          >
            Job created before
          </label>
          <div
            className="c21"
          >
            <div
              className="react-datepicker-wrapper"
            >
              <div
                className="react-datepicker__input-container"
              >
                <input
                  className="c22"
                  disabled={false}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Don't filter"
                  readOnly={false}
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <table
      className="c26"
      width="100%"
    >
      <thead
        className="c27"
      >
        <tr
          className="c28"
          cursor="auto"
        >
          <th
            className="c29"
            width="4%"
          >
            <label
              className="c17"
              color="black"
              fontSize={1}
              fontWeight="bold"
              width="100%"
            >
              <div
                className="c0 c30"
                cursor="auto"
                disabled={false}
              >
                <input
                  checked={false}
                  className="c31"
                  data-tag="masterCheckbox"
                  disabled={false}
                  onChange={[Function]}
                  size={20}
                  type="checkbox"
                />
                <svg
                  className="c32"
                  cursor="inherit"
                  data-name="checked"
                  fill="currentcolor"
                  height={20}
                  viewBox="0 0 24 24"
                  width={20}
                >
                  <path
                    d="M6 3h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3zm4 14l9-8.6L17.6 7 10 14.3l-3.6-3.5L5 12.2l5 4.8z"
                  />
                </svg>
                <svg
                  className="c32"
                  cursor="inherit"
                  data-name="empty"
                  fill="currentcolor"
                  height={20}
                  viewBox="0 0 24 24"
                  width={20}
                >
                  <path
                    d="M6 5c-.6 0-1 .4-1 1v12c0 .6.4 1 1 1h12c.6 0 1-.4 1-1V6c0-.6-.4-1-1-1H6zm0-2h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3z"
                  />
                </svg>
              </div>
            </label>
          </th>
          <th
            className="c33"
            onClick={[Function]}
          >
            Name
          </th>
          <th
            className="c33"
            onClick={[Function]}
          >
            State
          </th>
          <th
            className="c33"
            onClick={[Function]}
          >
            Application
          </th>
          <th
            className="c33"
            onClick={[Function]}
          >
            <svg
              className="c34"
              clipRule="evenodd"
              cursor="pointer"
              fill="currentcolor"
              fillRule="evenodd"
              height=".7em"
              mr=".4em"
              viewBox="0 0 18 24"
              width=".7em"
            >
              <path
                d="M9 24l9-10h-6V0H6v14h-6l9 10z"
              />
            </svg>
            Created at
          </th>
          <th
            className="c33"
            onClick={[Function]}
          >
            Expiration
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          className="c35"
          cursor="pointer"
        >
          <td
            className="c36 c37"
          >
            <div
              className="c0 c38"
              cursor="auto"
            >
              <label
                className="c17"
                color="black"
                fontSize={1}
                fontWeight="bold"
                width="100%"
              >
                <div
                  className="c0 c30"
                  cursor="auto"
                  disabled={false}
                >
                  <input
                    checked={false}
                    className="c31"
                    disabled={false}
                    onChange={[Function]}
                    size={20}
                    type="checkbox"
                  />
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="checked"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 3h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3zm4 14l9-8.6L17.6 7 10 14.3l-3.6-3.5L5 12.2l5 4.8z"
                    />
                  </svg>
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="empty"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 5c-.6 0-1 .4-1 1v12c0 .6.4 1 1 1h12c.6 0 1-.4 1-1V6c0-.6-.4-1-1-1H6zm0-2h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3z"
                    />
                  </svg>
                </div>
              </label>
            </div>
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            Thomas
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            <svg
              className="c40"
              clipRule="evenodd"
              color="#c00"
              cursor="inherit"
              fill="currentcolor"
              fillRule="evenodd"
              height={24}
              mr="8px"
              state="FAILURE"
              viewBox="0 0 25 24"
              width={24}
            >
              <path
                d="M24.002 2.001L22 .001 12 9.6l-10-9.6-2 2 9.6 10L.001 22l2 2 10-9.6 10 9.6 2-2-9.6-10 9.6-9.999z"
                fillRule="nonzero"
              />
            </svg>
             
            Failure
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            
             v
            
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            28/08/2018
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            N/A
          </td>
        </tr>
        <tr
          className="c35"
          cursor="pointer"
        >
          <td
            className="c36 c37"
          >
            <div
              className="c0 c38"
              cursor="auto"
            >
              <label
                className="c17"
                color="black"
                fontSize={1}
                fontWeight="bold"
                width="100%"
              >
                <div
                  className="c0 c30"
                  cursor="auto"
                  disabled={false}
                >
                  <input
                    checked={false}
                    className="c31"
                    disabled={false}
                    onChange={[Function]}
                    size={20}
                    type="checkbox"
                  />
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="checked"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 3h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3zm4 14l9-8.6L17.6 7 10 14.3l-3.6-3.5L5 12.2l5 4.8z"
                    />
                  </svg>
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="empty"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 5c-.6 0-1 .4-1 1v12c0 .6.4 1 1 1h12c.6 0 1-.4 1-1V6c0-.6-.4-1-1-1H6zm0-2h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3z"
                    />
                  </svg>
                </div>
              </label>
            </div>
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            Johnny
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            <svg
              className="c40"
              clipRule="evenodd"
              color="#c00"
              cursor="inherit"
              fill="currentcolor"
              fillRule="evenodd"
              height={24}
              mr="8px"
              state="FAILURE"
              viewBox="0 0 25 24"
              width={24}
            >
              <path
                d="M24.002 2.001L22 .001 12 9.6l-10-9.6-2 2 9.6 10L.001 22l2 2 10-9.6 10 9.6 2-2-9.6-10 9.6-9.999z"
                fillRule="nonzero"
              />
            </svg>
             
            Failure
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            
             v
            
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            29/08/2018
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            N/A
          </td>
        </tr>
        <tr
          className="c35"
          cursor="pointer"
        >
          <td
            className="c36 c37"
          >
            <div
              className="c0 c38"
              cursor="auto"
            >
              <label
                className="c17"
                color="black"
                fontSize={1}
                fontWeight="bold"
                width="100%"
              >
                <div
                  className="c0 c30"
                  cursor="auto"
                  disabled={false}
                >
                  <input
                    checked={false}
                    className="c31"
                    disabled={false}
                    onChange={[Function]}
                    size={20}
                    type="checkbox"
                  />
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="checked"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 3h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3zm4 14l9-8.6L17.6 7 10 14.3l-3.6-3.5L5 12.2l5 4.8z"
                    />
                  </svg>
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="empty"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 5c-.6 0-1 .4-1 1v12c0 .6.4 1 1 1h12c.6 0 1-.4 1-1V6c0-.6-.4-1-1-1H6zm0-2h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3z"
                    />
                  </svg>
                </div>
              </label>
            </div>
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            Fred
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            <svg
              className="c40"
              clipRule="evenodd"
              color="#c00"
              cursor="inherit"
              fill="currentcolor"
              fillRule="evenodd"
              height={24}
              mr="8px"
              state="FAILURE"
              viewBox="0 0 25 24"
              width={24}
            >
              <path
                d="M24.002 2.001L22 .001 12 9.6l-10-9.6-2 2 9.6 10L.001 22l2 2 10-9.6 10 9.6 2-2-9.6-10 9.6-9.999z"
                fillRule="nonzero"
              />
            </svg>
             
            Failure
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            
             v
            
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            30/08/2018
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            N/A
          </td>
        </tr>
        <tr
          className="c35"
          cursor="pointer"
        >
          <td
            className="c36 c37"
          >
            <div
              className="c0 c38"
              cursor="auto"
            >
              <label
                className="c17"
                color="black"
                fontSize={1}
                fontWeight="bold"
                width="100%"
              >
                <div
                  className="c0 c30"
                  cursor="auto"
                  disabled={false}
                >
                  <input
                    checked={false}
                    className="c31"
                    disabled={false}
                    onChange={[Function]}
                    size={20}
                    type="checkbox"
                  />
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="checked"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 3h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3zm4 14l9-8.6L17.6 7 10 14.3l-3.6-3.5L5 12.2l5 4.8z"
                    />
                  </svg>
                  <svg
                    className="c32"
                    cursor="inherit"
                    data-name="empty"
                    fill="currentcolor"
                    height={20}
                    viewBox="0 0 24 24"
                    width={20}
                  >
                    <path
                      d="M6 5c-.6 0-1 .4-1 1v12c0 .6.4 1 1 1h12c.6 0 1-.4 1-1V6c0-.6-.4-1-1-1H6zm0-2h12c1.7 0 3 1.3 3 3v12c0 1.7-1.3 3-3 3H6c-1.7 0-3-1.3-3-3V6c0-1.7 1.3-3 3-3z"
                    />
                  </svg>
                </div>
              </label>
            </div>
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            Nathan
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            <svg
              className="c41"
              clipRule="evenodd"
              color="#00C05A"
              cursor="inherit"
              fill="currentcolor"
              fillRule="evenodd"
              height={24}
              mr="8px"
              state="SUCCESS"
              viewBox="0 0 25 19"
              width={24}
            >
              <path
                d="M8 15L2 8.981l-2 2.02L8 19 24.001 2l-2.002-2L8 15z"
                fillRule="nonzero"
              />
            </svg>
             
            Success
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            
             v
            
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            03/09/2018
          </td>
          <td
            className="c36 c39"
            onClick={[Function]}
          >
            N/A
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
